index.html

```diff
@@ -37,7 +38,177 @@ <h1 class="header"><span>Hey there.
+                        <li><img src="img/javascript.png" alt="javascript"></li>
+                        <li><img src="img/html.png" alt="html"></li>
+                    </ul>
+                    <button class="btn" type="button"><span class="text">See project</span></button>
+                </div>
+            </div>
+            <section id="about-id">
+                <h1 class="header-2">About me</h1>
+
+
+                <p class="paragraph-1">I can help you build a product , <br>feature or website Look through <br>some of my work and experience! If <br>you like
+                    what you see and have a <br>project you need coded, <br> don’t hesitate to contact me.
+                </p>
+
+                <button class="btn-1" type="button"><span class="text">Get my resume</span></button>
+            </section>
+            <div class="grid-item-1">
+                <div>
+                    <div id="one"></div>
+                    <h4 class="header-3">Languages</h4>
+                    <ul class="image-container-1">
+                        <li><img src="img/js.png" alt="javascript"></li>
+                        <li><img src="img/rub.png" alt="ruby"></li>
+                        <li><img src="img/ht.png" alt="html"></li>
+                        <li><img src="img/cs.png" alt="css"></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="grid-item-1">
+                <div>
+                    <div id="two"></div>
+                    <h4 class="header-3">Frameworks</h4>
+                    <ul class="image-container-1">
+                        <li><img src="img/react.png" alt="React"></li>
+                        <li><img src="img/rails.png" alt="ruby"></li>
+                        <li><img src="img/rspec.png" alt="Rspec"></li>
+                        <li><img src="img/capy.png" alt="Capybara"></li>
+                        <li><img src="img/selenium.png" alt="Selenium"></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="grid-item-1">
+                <div>
+                    <div id="three"></div>
+                    <h4 class="header-3">Skills</h4>
+                    <ul class="image-container-1">
+                        <li><img src="img/database.png" alt="database"></li>
+                        <li><img src="img/git.png" alt="git"></li>
+                        <li><img src="img/cli.png" alt="cli"></li>
+                        <li><img src="img/web.png" alt="web"></li>
+                        <li><img src="img/api.png" alt="api"></li>
+                    </ul>
+                </div>
+            </div>
+            <div>
+                <h1 class="header-4">I'm always interested in hearing about new projects, so if you'd like to chat
+                    please get in touch.</h1>
+            </div>
+            <div>
+                <form class="form">
+                    <div class="form-group">
```

Comment:
> Collect data by using the [Formspree service](https://formspree.io/html).

- [ ] According to the project requirement, you are meant to have a post method in your form to submit the to form spree. I'll suggest you follow [this link](https://formspree.io/html), create a form spree account and get your post request URL, then add it to your form element. :smile: :wave: